tests/func/test_ls.py

```diff
@@ -57,16 +57,44 @@ def test_ls_repo(tmp_dir, dvc, scm):
     match_files(
         files,
         (
-            (".gitignore",),
-            ("README.md",),
-            ("structure.xml.dvc",),
-            ("model",),
-            ("data",),
-            ("structure.xml",),
+            ((".gitignore",), False),
+            (("README.md",), False),
+            (("structure.xml.dvc",), False),
+            (("model",), False),
+            (("data",), False),
+            (("structure.xml",), True),
         ),
     )
 
 
+def test_ls_repo_with_color(tmp_dir, dvc, scm, mocker, caplog):
+    import logging
+    from dvc.cli import parse_args
+
+    tmp_dir.scm_gen(FS_STRUCTURE, commit="init")
+    tmp_dir.dvc_gen(DVC_STRUCTURE, commit="dvc")
+
+    os.environ["LS_COLORS"] = "rs=0:di=01;34:*.xml=01;31:*.dvc=01;33:"
```

Comment:
This is bad, you need to use mocker to temporarily set env. Otherwise you are affecting all the other tests.